_data/projects/Devdojo_tails.yml

```diff
@@ -0,0 +1,13 @@
+name: tails
+desc: Tails is a (no-config) copy'n paste library of hand-crafted templates and components built in TailwindCSS.
+      It's pretty simple to use any of these components or templates in your project. Any component you wish to use can be copied and pasted into your own project. Visit the Components page to view the components and view the source code. Additionally, you can view the Templates and simply copy and paste these templates into your own project.
+site: https://github.com/thedevdojo/tails
+tags:
+  - web
+  - javascript
+  - css
+  - education
+  - learning
+upforgrabs:
+  name: help wanted
+  link: https://github.com/thedevdojo/tails/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22
```

Comment:
@ganapathi12 Please use this instead https://github.com/thedevdojo/tails/labels/help%20wanted